roles/openshift_node/templates/bootstrap.yml.j2

```diff
@@ -1,22 +1,22 @@
 {% raw %}
 #!/usr/bin/ansible-playbook
 ---
-- hosts: localhost
-  gather_facts: yes
-  vars:
-    origin_dns:
-      file: /etc/dnsmasq.d/origin-dns.conf
-      lines:
-      - regex: ^listen-address
-        state: present
-        line: "listen-address={{ ansible_default_ipv4.address }}"
-    node_dns:
-      file: /etc/dnsmasq.d/node-dnsmasq.conf
-      lines:
-      - regex: "^server=/in-addr.arpa/127.0.0.1$"
-        line: server=/in-addr.arpa/127.0.0.1
-      - regex: "^server=/cluster.local/127.0.0.1$"
-        line: server=/cluster.local/127.0.0.1
+	- hosts: localhost
```

Comment:
Why the extra indent?

Reply:
Which indent do you mean? In the source there is no additional indent, see:
https://github.com/reneklootwijk/openshift-ansible/blob/release-3.9/roles/openshift_node/templates/bootstrap.yml.j2
`
	      lines:
	      - regex: "^server=/in-addr.arpa/127.0.0.1$"
	        line: server=/in-addr.arpa/127.0.0.1
	      - regex: "^server=/{{openshift.common.dns_domain}}/127.0.0.1$"
	        line: server=/{{openshift.common.dns_domain}}/127.0.0.1
`

Reply:
It looks like an [extra tab](https://i.imgur.com/9kibJRo.png) added in this PR

Reply:
Sorry, you are absolutely right. I committed a fix.